noxfile.py

```diff
@@ -54,15 +54,13 @@
 current_python = f"{sys.version_info.major}.{sys.version_info.minor}"
 nox.options.sessions = [f"tests-{current_python}(skipslow)"]
 
-nox.options.default_venv_backend = "uv|virtualenv"
+nox.options.default_venv_backend = "uv"
```

Comment:
Because there were a whole bunch of Nox sessions that installed and depended upon `uv`, it makes sense to have `uv` be the only backend.